### ISSUE TITLE
Server crashes on getting an invalid date value

### ISSUE THREAD

**Issue by ashwin95r** [dgraph-io/dgraph#452]
Logs:
```
Query received: {
  debug(_xid_: m.0bxtg) {
    type.object.name.en
    film.director.film @filter(geq("film.film.initial_release_date", "197-01-01")) {
      film.film.initial_release_date
      type.object.name.en
    }
  }
}
Xid: m.0bxtg Uid: 14685953405111677952
Query parsed
attr: film.director.film groupId: 1
Sample value for attr: film.director.film Val: 
attr: type.object.name.en groupId: 1
Sample value for attr: type.object.name.en Val: Tom Hanks
Zero uids for "type.object.name.en". Num attr children: 0
attr: film.film.initial_release_date groupId: 1
2017/01/06 17:05:20 error.go:44: parsing time "197-01-01" as "2006": cannot parse "01-01" as "2006"
github.com/dgraph-io/dgraph/x.Wrap
        /home/ubuntu/go/src/github.com/dgraph-io/dgraph/x/error.go:84
github.com/dgraph-io/dgraph/x.Check
        /home/ubuntu/go/src/github.com/dgraph-io/dgraph/x/error.go:44
github.com/dgraph-io/dgraph/worker.convertValue
        /home/ubuntu/go/src/github.com/dgraph-io/dgraph/worker/task.go:85
github.com/dgraph-io/dgraph/worker.processTask
        /home/ubuntu/go/src/github.com/dgraph-io/dgraph/worker/task.go:115
github.com/dgraph-io/dgraph/worker.ProcessTaskOverNetwork
        /home/ubuntu/go/src/github.com/dgraph-io/dgraph/worker/task.go:48
github.com/dgraph-io/dgraph/query.ProcessGraph
        /home/ubuntu/go/src/github.com/dgraph-io/dgraph/query/query.go:513
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:2086
```

**Comment by manishrjain** [comment]
@ashwin95r -- Can you fix this, merge it on the 0.7.1 branch and then push it to the server? We don't want crashes; so let's do this asap.

**Comment by ashwin95r** [comment]
Fixed.
